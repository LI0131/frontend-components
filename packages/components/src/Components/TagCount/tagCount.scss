.ins-c-tag-count {
    color: var(--pf-global--icon--Color--light);
    display: flex !important;
    align-content: center;
}

.ins-c-tag__text {
    margin-left: 10px;
}