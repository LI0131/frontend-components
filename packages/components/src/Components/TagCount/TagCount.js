import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import './tagCount.scss';

export default class TagCount extends React.Component
{
    constructor(props) {
        super(props);
    }
    render() {
        return (
            <div className="ins-c-tagCount">
                <i className="fas fa-tag"></i>
                <p>{this.props.count}</p>
            </div>
        )
    }
}