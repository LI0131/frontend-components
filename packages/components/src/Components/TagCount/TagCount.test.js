import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import TagCount from './TagCount';

describe('TagCount component', () => {
    it('should render a tag count with no value', () => {
        const wrapper = shallow(<TagCount/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render a tag count of 11', () => {
        const wrapper = shallow(<TagCount systemName="paul.localhost.com"/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    });
})
