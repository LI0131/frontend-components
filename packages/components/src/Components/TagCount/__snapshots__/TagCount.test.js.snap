// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagCount component should render 1`] = `
<div
  className="ins-c-tagCount"
>
  <i
    className="fas fa-tag"
  />
  <p />
</div>
`;

exports[`TagCount component should render center Spinner 1`] = `
<div
  className="ins-c-tagCount"
>
  <i
    className="fas fa-tag"
  />
  <p>
    44
  </p>
</div>
`;
